occlum/run_test_case.sh

```diff
@@ -0,0 +1,112 @@
+
+#!/bin/bash
+TEST_CASE_DIR=/opt/occlum/gvisor_syscall_tests
+BLOCKLIST_DIR=../blocklist
+NGO_BLOCKLIST_DIR=../ngo_blocklist
+FILTER=""
+NGO_FILTER=""
+VERSION=""
+num1=0
```

Comment:
Please rename the variable with a more precise name.

Reply:
Modification complete